packages/calcite-components/src/components/tab-nav/tab-nav.e2e.ts

```diff
@@ -37,7 +37,6 @@ describe("calcite-tab-nav", () => {
     const activeEventSpy = await page.spyOnEvent("calciteTabChange");
     const firstTabTitle = await page.find("calcite-tab-title");
 
-    firstTabTitle.setProperty("selected", true);
```

Comment:
We still need this to ensure it doesn’t emit when selecting tab titles programmatically, right?